src/TransitionViz.scss

```diff
@@ -72,3 +72,17 @@
   }
   padding: 0rem 0.5rem 0.5rem;
 }
+
+[data-viz='transition-description'] {
```

Comment:
Is this used?

Reply:
used in https://github.com/statelyai/xstate-viz/pull/338/files#diff-604a3c1c93422cac123264644254df661894348c16b9cfa9afea3cfad1633681R169